### PR TITLE
Add planar support surfaces to RigidBodySupportState

### PR DESCRIPTION
**Pull request by kuindersma**

### CODE REVIEW

File: lcmtypes/lcmt_support_data.lcm [hunk 1]
```diff
@@ -11,7 +11,8 @@ struct lcmt_support_data
 
 	boolean support_logic_map[4];
 
-	double mu;
+	boolean use_support_surface; 
+	float support_surface[4];
```
Comment (RussTedrake) [inline lcmtypes/lcmt_support_data.lcm]: add inline doc?  what 4 numbers are you using to represent the plane?

File: systems/controllers/controlUtil.h [hunk 2]
```diff
@@ -33,8 +33,9 @@ typedef struct _support_state_element
 {
   int body_idx;
   std::vector<Vector4d, aligned_allocator<Vector4d>> contact_pts;
-  int contact_surface;
   bool support_logic_map[4];
+  bool use_support_surface;
+  Vector4d support_surface;
```
Comment (RussTedrake) [inline systems/controllers/controlUtil.h]: document it here, too?

File: systems/plants/RigidBodySupportState.m [hunk 3]
```diff
@@ -8,27 +8,32 @@
     contact_groups; % cell array of cell arrays of contact group strings, 1 for each body
     num_contact_pts;  % convenience array containing the desired number of
                       %             contact points for each support body
-    contact_surfaces; % int IDs either: 0 (terrain), -1 (any body in bullet collision world)
-                      %             or (1:num_bodies) collision object ID
-  end
+    use_support_surface; % logical vector with the same length as bodies
+    support_surface; % 4-vector describing a support surface: [v; b] such that v' * [x;y;z] + b == 0
```
Comment (RussTedrake) [inline systems/plants/RigidBodySupportState.m]: ah.  there it is. :)

**Comment by RussTedrake** [comment]
apart from asking for a little more documentation, it looks good to me.  went through it fairly carefully.

File: systems/controllers/controlUtil.h (hunk 2 above)
Comment (RussTedrake) [inline systems/controllers/controlUtil.h]: wow.  the windows compiler caught you here.  (thank you MSVC!).  

If you have a Vector4d in your class, then you need to add 
```
public:
  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
```
http://eigen.tuxfamily.org/dox-devel/group__TopicStructHavingEigenMembers.html
to your class.  I should have caught that one.

**Comment by RussTedrake** [comment]
some other msvc warnings that you could clean up as you touch this again. 

```
..\..\..\systems\controllers\controlUtil.cpp(71): warning C4018: '<' : signed/u\
nsigned mismatch [C:\cygwin\tmp\drake-precompiled\drake-16023\pod-build\systems\
\controllers\drakeControlUtil.vcxproj]^M
..\..\..\systems\controllers\controlUtil.cpp(255): warning C4018: '<' : signed/\
unsigned mismatch [C:\cygwin\tmp\drake-precompiled\drake-16023\pod-build\system\
s\controllers\drakeControlUtil.vcxproj]^M
..\..\..\systems\controllers\controlUtil.cpp(273): warning C4018: '<' : signed/\
unsigned mismatch [C:\cygwin\tmp\drake-precompiled\drake-16023\pod-build\system\
s\controllers\drakeControlUtil.vcxproj]^M
..\..\..\systems\controllers\controlUtil.cpp(405): warning C4018: '<' : signed/\
unsigned mismatch [C:\cygwin\tmp\drake-precompiled\drake-16023\pod-build\system\
s\controllers\drakeControlUtil.vcxproj]^M
..\..\..\systems\controllers\controlUtil.cpp(438): warning C4018: '<' : signed/\
unsigned mismatch [C:\cygwin\tmp\drake-precompiled\drake-16023\pod-build\system\
s\controllers\drakeControlUtil.vcxproj]^M
```

**Comment by kuindersma** [comment]
Is this good to go now?

**Comment by RussTedrake** [comment]
Windows build servers must be stuck.  will resolve and merge shortly

**Pull request merged**